src/messages/pt.json

```diff
@@ -273,26 +367,24 @@
   "known_issues_result.empty": "Nenhum resultado correspondente. Tente outros filtros.",
   "known_issues_date.created": "Criado em",
   "known_issues_date.updated": "Atualizado em",
-  "tutorial_and_solutions_page.title": "Tutoriais",
-  "tutorial_and_solutions_page.description": "Acesse a documentação de todas os módulos da plataforma e tire suas dúvidas.",
+  "tutorials_page.title": "Tutoriais hub",
+  "tutorials_page.description": "Navegue pelos nossos tutoriais para melhorar sua compreensão da plataforma.",
   "date_text.created": "Publicado em",
   "date_text.updated": "Última atualização em",
   "announcements_page_result.empty": "Nenhum resultado correspondente.",
   "announcements_page_search.placeholder": "Pesquise em Comunicados...",
   "search_result.empty": "Nenhum resultado correspondente. Tente outros filtros.",
   "faq_filter.title": "Produtos",
-  "faq_page.results_found": "resultados encontrados",
   "sitemap_page.title": "Mapa do site",
   "sitemap_page_section_additional_resources.title": "Recursos Adicionais",
   "sitemap_page_section_additional_resources.support_rules": "Diretrizes de Suporte",
-  "sitemap_page_section_additional_resources.known_issues": "Problemas conhecidos",
-  "sitemap_page_section_additional_resources.faq": "Perguntas frequentes",
+  "sitemap_page_section_additional_resources.known_issues": "Problemas Conhecidos",
+  "sitemap_page_section_additional_resources.faq": "Perguntas Frequentes",
   "sitemap_page_section_additional_resources.announcements": "Comunicados",
-  "sitemap_page_section_additional_resources.dev_portal": "Developer portal",
+  "sitemap_page_section_additional_resources.dev_portal": "Developers Portal",
```

Comment:
None of this should be changed

Reply:
This change is due to another modification occurring after the creation of this branch. I will restore the original information.